metadata.rb

```diff
@@ -1,17 +1,18 @@
-maintainer       "Alistair Stead"
-maintainer_email "alistair@inviqa.com"
-license          "Apache 2.0"
-description      "Installs/Configures chef-varnish"
-name             "chef-varnish"
+maintainer       'Alistair Stead'
+maintainer_email 'alistair@inviqa.com'
+license          'Apache 2.0'
+description      'Installs/Configures chef-varnish'
+name             'chef-varnish'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "1.0.6"
+version          '2.0.0'
 
-depends "apt"
-depends "yum"
+depends 'apt'
+depends 'yum'
 
-%w{redhat centos scientific fedora debian ubuntu arch freebsd amazon}.each do |os|
+depends 'packagecloud', '~> 0.3.0'
+
+%w(redhat centos scientific fedora debian ubuntu arch freebsd amazon).each do |os|
```

Comment:
should we remove arch/freebsd? Amazon also it's own platform_family but is supported as "rpm"